### PR TITLE
Update Graphics, move RLWR indications to MPD draw system

### PR DESCRIPTION
**Pull request by BradMick**

### CODE REVIEW

File: @AH-64D Apache Longbow/Addons/fza_ah64_mpd/functions/page/fn_pageTSDDraw.sqf [hunk 1]
```diff
@@ -174,6 +174,87 @@ if (count _fcrTargets > 0) then {
...
+
+private _aseObjects  = _heli getVariable "fza_ah64_ase_rlwrObjects";
+private _radius      = 1.00;
+{
+    _x params ["_state", "_bearing", "_classification"];
+    private _ident      = "";
+    private _unitStatus = "";
+    private _unitType   = "";
+    //Unit type
+    switch (_classification) do {
```
Comment (mattysmith22) [inline @AH-64D Apache Longbow/Addons/fza_ah64_mpd/functions/page/fn_pageTSDDraw.sqf:187]: Can we deduplicate the code between `pageASEDraw` and `pageTSDDraw` by adding a new function?

File: @AH-64D Apache Longbow/Addons/fza_ah64_mpd/pages/ase_draw.hpp [hunk 2]
```diff
@@ -429,6 +429,7 @@ class ase_draw {
     };
 };
 
+/*
```
Comment (mattysmith22) [inline @AH-64D Apache Longbow/Addons/fza_ah64_mpd/pages/ase_draw.hpp:432]: If we no longer need this, can we just remove?

**Review by mattysmith22 — CHANGES_REQUESTED** [review]

File: @AH-64D Apache Longbow/Addons/fza_ah64_mpd/pages/ase_draw.hpp [hunk 3]
```diff
@@ -429,6 +429,7 @@ class ase_draw {
...
 class ase_threats_gnd {
     condition = C_COND(C_NOT(C_MPD_USER(MFD_IND_ASE_RLWR_PWR)));
     color[] = {1,1,0,1};
```
Comment (Ollieollieolllie) [inline @AH-64D Apache Longbow/Addons/fza_ah64_mpd/pages/ase_draw.hpp:435]: You would need to leave at least the Msl drawing for the Uk variant unless you also plan to add that the same way

**Review by Ollieollieolllie — CHANGES_REQUESTED** [review]

File: @AH-64D Apache Longbow/Addons/fza_ah64_mpd/functions/page/fn_pageTSDDraw.sqf (hunk 1 above)
Comment (BradMick) [reply @AH-64D Apache Longbow/Addons/fza_ah64_mpd/functions/page/fn_pageTSDDraw.sqf:187]: Not sure, they're two unique implementations. The ASE page is fixed, whereas the TSD page has to account for centered vs. decentered.

File: @AH-64D Apache Longbow/Addons/fza_ah64_mpd/pages/ase_draw.hpp (hunk 3 above)
Comment (BradMick) [reply @AH-64D Apache Longbow/Addons/fza_ah64_mpd/pages/ase_draw.hpp:435]: We're not mixing the two.

Comment (Ollieollieolllie) [reply @AH-64D Apache Longbow/Addons/fza_ah64_mpd/pages/ase_draw.hpp]: So you plan to add this to be drawn aswell then?

File: @AH-64D Apache Longbow/Addons/fza_ah64_mpd/FzaIconTypes.hpp [hunk 4]
```diff
@@ -6,6 +6,7 @@
 #define MPD_ICON_TYPE_F 5
 #define MPD_ICON_TYPE_G 6
 #define MPD_ICON_TYPE_H 7
+#define MPD_ICON_TYPE_I 8
```
Comment (mattysmith22) [inline @AH-64D Apache Longbow/Addons/fza_ah64_mpd/FzaIconTypes.hpp:9]: Are these even used anymore actually?

File: @AH-64D Apache Longbow/Addons/fza_ah64_mpd/functions/page/fn_pageASEDraw.sqf [hunk 5]
```diff
@@ -45,4 +46,83 @@ _msn_equip_British = _heli animationPhase "msn_equip_british";
...
+        };
+        case ASE_TRK: {
+            _unitStatus = "ACQ_TRK";
+        };
+        case ASE_LNC: {
+            _unitStatus = "LNC";
+        };
+    };
+
+    _ident           = (["RLWR", _unitType,_unitStatus]) joinString "_";
```
Comment (mattysmith22) [inline @AH-64D Apache Longbow/Addons/fza_ah64_mpd/functions/page/fn_pageASEDraw.sqf:124]: Could you unity the creation of this ident between TSD and ASE into a single function, as it looks identical?

File: @AH-64D Apache Longbow/Addons/fza_ah64_mpd/functions/page/fn_pageTSDDraw.sqf [hunk 6]
```diff
@@ -174,6 +174,121 @@ if (count _fcrTargets > 0) then {
...
+        };
+        case ASE_LNC: {
+            _unitStatus = "LNC";
+        };
+    };
+
+    _ident           = (["RLWR", _unitType,_unitStatus]) joinString "_";
+
+    private _ctrX = 0.5;  private _ctrY = 0.75 - 0.25 * (_persistState get "ctr");
+    private _minX = 0.23; private _maxX = 0.77;
```
Comment (mattysmith22) [inline @AH-64D Apache Longbow/Addons/fza_ah64_mpd/functions/page/fn_pageTSDDraw.sqf:252]: This feels like a useful function to split out. Could you make a function - possibly `fza_mpd_fnc_projectBearingToScreenEdge` or something?

**Review by mattysmith22 — CHANGES_REQUESTED** [review]
Looks great! Just a couple more changes and we're good

**Review by mattysmith22 — APPROVED** [review]

**Review by Ollieollieolllie — APPROVED** [review]

**Pull request merged**